src/main/java/uk/gov/hmcts/divorce/legaladvisor/service/printer/AwaitingAmendedApplicationPrinter.java

```diff
@@ -79,15 +87,22 @@ public void sendLetters(final CaseData caseData, final Long caseId, final Applic
         }
     }
 
-    private List<Letter> awaitingAmendedApplicationLetters(final CaseData caseData) {
+    private Boolean isPaperCaseAndApplicantRepresented(final CaseData caseData, final Applicant applicant) {
```

Comment:
Worth checking everywhere this is done - if we reach a printer class it's already confirmed offline / paper case

Reply:
Good point - I was being overly defensive.  Resolved.